tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -199,12 +211,12 @@ private void ExecuteFunction(String Function)
                     {
                         String newini = ini.Substring(0, pos + 9);
                         newini += SelectedRenderer;
-                        newini += ini.Substring(pos + 10, ini.Length - pos - 10);
+                        newini += ini.Substring(pos + 11, ini.Length - pos - 11);
                         File.WriteAllText(AppDomain.CurrentDomain.BaseDirectory + "GSDumpGSDXConfigs\\" + GSDXName + "\\inis\\gsdx.ini", newini);
                     }
                     else
                     {
-                        File.WriteAllText(AppDomain.CurrentDomain.BaseDirectory + "GSDumpGSDXConfigs\\" + GSDXName + "\\inis\\gsdx.ini", ini + Environment.NewLine + "Renderer=" + SelectedRenderer);
+                        File.WriteAllText(AppDomain.CurrentDomain.BaseDirectory + "GSDumpGSDXConfigs\\" + GSDXName + "\\inis\\gsdx.ini", ini + "[Settings]" + Environment.NewLine + "Adapter=default" + Environment.NewLine + "Renderer=" + SelectedRenderer + Environment.NewLine);
```

Comment:
The `Adapter=default` part is probably unnecessary.

Reply:
It isn't. Added it just in case. [Settings] is tho , as without it it will conflict when changing Gsdx settings.